### PR TITLE
feature/193 making the parameter naming consistent.

### PR DESCRIPTION
**Pull request by DimitriDiantos**
Make parameter naming consistent: upper / lower case.

### CODE REVIEW

File: de.dlr.sc.virsat.model.extension.cef.ui/src/de/dlr/sc/virsat/model/extension/cef/ui/templates/DLRCEFStudyCommandHelper.java [hunk 1]
```diff
@@ -42,6 +42,15 @@
  */
 public class DLRCEFStudyCommandHelper {
 	
+	public static final String POWER_PARAMETERS = "powerParameters";
+	public static final String MASS_PARAMETERS = "massParameters";
+	public static final String SYSTEM_PARAMETERS = "systemParameters";
+	public static final String EQUIPMENT_MASS_PARAMETERS = "EquipmentMassParameters";
```
Comment (dellerDLR) [inline de.dlr.sc.virsat.model.extension.cef.ui/src/de/dlr/sc/virsat/model/extension/cef/ui/templates/DLRCEFStudyCommandHelper.java:48]: Shouldn't all parameter start with a small or a big letter?
What do you prefer @franzTobiasDLR ?

**Review by dellerDLR — CHANGES_REQUESTED** [review]

File: de.dlr.sc.virsat.model.extension.cef.ui/src/de/dlr/sc/virsat/model/extension/cef/ui/templates/DLRCEFStudyCommandHelper.java (hunk 1 above)
Comment (franzTobiasDLR) [reply de.dlr.sc.virsat.model.extension.cef.ui/src/de/dlr/sc/virsat/model/extension/cef/ui/templates/DLRCEFStudyCommandHelper.java:48]: Classes / CAs upper case, properties lower case

File: de.dlr.sc.virsat.model.extension.cefx.test/src/de/dlr/sc/virsat/model/extension/cefx/migrator/Migrator1v2Test.java [hunk 2]
```diff
@@ -0,0 +1 @@
+
```
Comment (franzTobiasDLR) [inline de.dlr.sc.virsat.model.extension.cefx.test/src/de/dlr/sc/virsat/model/extension/cefx/migrator/Migrator1v2Test.java:1]: Why do you commit an empty class? Where is the migrator gone? Without the class it won't work...

File: de.dlr.sc.virsat.model.extension.cefx/src/de/dlr/sc/virsat/model/extension/cefx/migrator/Migrator1v2.java [hunk 3]
```diff
@@ -0,0 +1 @@
+
```
Comment (franzTobiasDLR) [inline de.dlr.sc.virsat.model.extension.cefx/src/de/dlr/sc/virsat/model/extension/cefx/migrator/Migrator1v2.java:1]: Also here, the file shouldn't be empty

**Review by franzTobiasDLR — CHANGES_REQUESTED** [review]

**Review by franzTobiasDLR — APPROVED** [review]
Looking good from my side! Thanks!

**Pull request closed without merging**